src/routes/Home.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
Can you rename the routes folder to pages. Trying to some what follow [atomic design](https://bradfrost.com/blog/post/atomic-web-design/). At least as far as organizing components.